src/backend/aspen/workflows/import_gisaid_isls/save.py

```diff
@@ -0,0 +1,69 @@
+# flake8: noqa: E711
+# Doing a double-equals comparison to None is critical for the statements
+# that use it to compile to the intended SQL, which is why tell flake8 to
+# ignore rule E711 at the top of this file
+
+import click
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.sql.expression import literal_column
+
+from aspen.config.config import Config
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+from aspen.database.models import Accession, AccessionType, GisaidMetadata, Sample
+
+
+def save():
+    config = Config()
+    interface: SqlAlchemyInterface = init_db(get_db_uri(config))
+
+    with session_scope(interface) as session:
+        # Note: the syntax of the literal_column() is purposefully done so the resulting
+        # PostgreSQL expression will have single-quotes around it, since we are
+        # SELECTing on a string literal (this is done in order to give every row in this subquery
+        # the value 'GISAID_ISL' in the 'accession_type' column.)
+        subquery = (
+            sa.select(
+                Sample.id,
+                literal_column(f"'{AccessionType.GISAID_ISL.value}'").label(
+                    "accession_type"
+                ),
+                GisaidMetadata.gisaid_epi_isl,
+            )
+            .select_from(Sample)
+            .join(GisaidMetadata, Sample.public_identifier == GisaidMetadata.strain)
+            .subquery()
+        )
+
+        insert_accessions_stmt = insert(Accession.__table__).from_select(
```

Comment:
This query is identical to the one in your migration, ya?

Reply:
yep